python/cudf/cudf/core/column/string.py

```diff
@@ -114,7 +114,7 @@ def cat(self, others=None, sep=None, na_rep=None):
 
         Parameters
         ----------
-            others : Series or List of str
+            others : Series, StringMethods or List of str
```

Comment:
This actually doesn't operate on the `StringMethods` object since it raises an exception. We should update this parameter's docstring accordingly.

Reply:
ohh right! will update it.